components/charts/WordCloud.vue

```diff
@@ -65,9 +80,57 @@ export default {
     this.series.accuracy = 5;
     this.updateGraph();
   },
-  beforeDestroy: function () {
+  beforeDestroy: function() {
     this.chart.dispose();
   },
+  methods: {
```

Comment:
A initial comment what the function is doing would be nice to have: 

Something like: Filter out words from exclude list